xml/System.Threading/ThreadPool.xml

```diff
@@ -1267,13 +1268,13 @@
         <Parameter Name="preferLocal" Type="System.Boolean" Index="2" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TState">To be added.</typeparam>
-        <param name="callBack">To be added.</param>
-        <param name="state">To be added.</param>
-        <param name="preferLocal">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TState">The type of elements of <paramref name="state" />.</typeparam>
+        <param name="callBack">An <see cref="T:System.Action`1" /> representing the method to execute.</param>
```

Comment:
```suggestion
        <param name="callBack">A delegate representing the method to execute.</param>
```
The delegate type appears immediately above the description, so it's not necessary to duplicate it.